src/dc_simplify.rs

```diff
@@ -5,16 +5,30 @@ use crate::x::*;
 #[derive(Copy, Clone)]
 pub struct Simplify {
     pub is_forwarded: bool,
-    pub is_cut_at_begin: bool,
-    pub is_cut_at_end: bool,
+}
+
+/// Return index of footer line in vector of message lines, or vector length if
+/// no footer is found.
+///
+/// Also return whether not-standard (rfc3676, §4.3) footer is found.
+fn find_message_footer(lines: &[&str]) -> (usize, bool) {
+    for ix in 0..lines.len() {
+        let line = lines[ix];
+
+        // XXX: Why "--   " (two spaces) is considered standard?
```

Comment:
Move comment from this line here instead:
https://github.com/deltachat/deltachat-core/blob/e4ce281dc2c8cb1d0765f0e2e73f7846cbba3faa/src/dc_simplify.c#L119